src/feinstaub-api.js

```diff
@@ -1,40 +1,39 @@
 // const URL = 'https://api.luftdaten.info/static/v2/data.dust.min.json'
 // const URL = 'https://api.luftdaten.info/v1/filter/area=48.800000,9.200000,50'
 // const URL = 'https://api.airquality.codeforafrica.org/v2/sensors/?type=PPD42NS,HPM,PMS1003,PMS3003,PMS5003,PMS6003,PMS7003,SDS021,SDS011'
-const URL = "https://api.sensors.africa/v2/nodes/";
+const URL = 'http://api.sensors.africa/v2/nodes/?format=json'
+// const URL = "http://staging.api.sensors.africa/v2/nodes/";
 
-//const URL = "http://staging.api.sensors.africa/v2/nodes/";
-
-import _ from "lodash";
-import "whatwg-fetch";
+import _ from 'lodash'
+import 'whatwg-fetch'
 
 let api = {
-  fetchNow() {
-    return fetch(URL).then(response => response.json());
+  fetchNow () {
```

Comment:
Not sure about this space between `function` and `()`. Aren't we using [AirBnB style](https://github.com/airbnb/javascript#functions--signature-spacing)?